ultimaker_rapidrinse_175.xml.fdm_material

```diff
@@ -28,7 +29,7 @@
 
         <!-- Define as soluble support material -->
         <cura:setting key="material_is_support_material">True</cura:setting>
-        <cura:setting key="prime_tower_enable">=extruder_nr==support_extruder_nr</cura:setting>
```

Comment:
This change is not OK. It is a much larger formula. See PP-401.

Reply:
ya copy past error, PR is still in draft. Will fix this later

Reply:
https://github.com/Ultimaker/fdm_materials/pull/273/commits/5fe8bdd965c427082b247312220ac185a1697846